holmes/plugins/toolsets/infrainsights/comprehensive_kafka_connect_toolset.py

```
@@ -0,0 +1,1077 @@
+                    data={
+                        "message": f"Connector {connector_name} created successfully",
+                        "connector_info": result_data
+                    },
+                    params=params
+                )
+            else:
+                error_msg = f"Failed to create connector: {response.status_code}"
+                try:
+                    error_data = response.json()
+                    error_msg += f" - {error_data.get('message', 'Unknown error')}"
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)